wodles/aws/aws_s3.py

```diff
@@ -2544,14 +2558,19 @@ def _key_is_old(self, file_date: datetime or None, last_key_date: datetime or No
         return False
 
     def iter_files_in_bucket(self, aws_account_id: str = None, aws_region: str = None):
+        if aws_account_id is None:
+            aws_account_id = self.aws_account_id
+
         try:
             bucket_files = self.client.list_objects_v2(**self.build_s3_filter_args(aws_account_id, aws_region,
                                                                                    custom_delimiter='-'))
             while True:
                 if 'Contents' not in bucket_files:
-                    debug(f"+++ No logs to process in bucket: {aws_account_id}/{aws_region}", 1)
+                    self._print_not_logs_to_prcess_message(self.bucket, aws_account_id, aws_region)
```

Comment:
Typo in the method's call
```suggestion
                    self._print_not_logs_to_process_message(self.bucket, aws_account_id, aws_region)
```